card/mod/05_standard/set/right/account.rb

```diff
@@ -74,20 +74,20 @@ def authenticate_by_token val
 event :set_default_salt, on: :create, before: :process_subcards do
   salt = Digest::SHA1.hexdigest "--#{Time.now.to_s}--"
   Env[:salt] = salt # HACK!!! need viable mechanism to get this to password
-  subcards["+#{Card[:salt].name}"] ||= { content: salt }
+  add_subfield :salt, content: salt
 end
 
 event :set_default_status, on: :create, before: :process_subcards do
   default_status = ( Auth.needs_setup? ? 'active' : 'pending' )
-  subcards["+#{Card[:status].name}"] = { content: default_status }
+  add_subfield :status, content: default_status
 end
 
 def confirm_ok?
   Card.new( type_id: Card.default_accounted_type_id ).ok? :create
 end
 
-event :generate_confirmation_token, on: :create, before: :process_subcards, when: proc{ |c| c.confirm_ok? } do
-  subcards["+#{Card[:token].name}"] = {content: generate_token }
+event :generate_confirmation_token, :on=>:create, :before=>:process_subcards, :when=>proc{ |c| c.confirm_ok? } do
```

Comment:
Line is too long. [113/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.<br>Space missing to the left of {.